models/ticket_history/int_zendesk__field_history_pivot.sql

```diff
@@ -1,3 +1,5 @@
+-- depends_on: {{ ref('stg_zendesk__ticket_field_history') }}
```

Comment:
do we need this here?

Reply:
We need this here in order for the run_query statement to properly reference the ticket_field_history model